functions/Get-DbaDbMailProfile.ps1

```diff
@@ -88,17 +94,17 @@ function Get-DbaDbMailProfile {
                 $profiles = $mailserver.Profiles
 
                 if ($Profile) {
-                    $profiles = $profiles | Where-Object Name -in $Profile
+                    $profiles = $profiles | Where-Object Name -In $Profile
```

Comment:
We like to use the all lowercase version even if not "correct".

Reply:
I think that was corrected by Invoke-DbatoolsFormatter as I've spotted that after running the command. Happy to change it back to lowercases, but it may come back.

Reply:
If it does come back, ask @niphlod for help - he is our formatting expert.

Reply:
just for the sake of the discussion, operators with camelcase do happen if there's a recent version of scriptanalyzer installed, as "the gods" (creators of scriptanalyzer) prefer the canonical operator namecase convention. We try our best to load the correct one and warn about mismatches https://github.com/sqlcollaborative/dbatools/blob/90474c562c7696c8957c38e8becbb4ca408f53a8/functions/Invoke-DbatoolsFormatter.ps1#L47 but prolly in some envs it still behaves incorrectly.

Reply:
Oh, ok. That makes sense, wasn't aware of that. I think I've imported the latest version. When have a chance will confirm that.